_templates/legacy.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * _templates/legacy.php
+ * Holds all legacy browser messages warnings
+ */
+
+$l10n->set_domain('layout');
+
+$user_agent = $_SERVER['HTTP_USER_AGENT'];
+
+$is_HTTP = (empty($_SERVER['HTTPS']) || $_SERVER['HTTPS'] !== 'on');
+$is_old_IE = preg_match('/MSIE [2-10]/i', $user_agent);
+
+if (getenv('PHPENV') === 'production' && $is_HTTP) {
```

Comment:
Not on our production servers, but I felt it was needed in case someone else trys to host, or if we setup a new server that does not have the certificate generated yet.